core/common/src/main/java/alluxio/security/authorization/AclEntry.java

```diff
@@ -199,20 +257,38 @@ public static AclEntry fromCliStringWithoutPermissions(String stringEntry) {
     }
     List<String> components = Arrays.stream(stringEntry.split(":")).map(String::trim).collect(
         Collectors.toList());
-    if (components.size() < 1 || components.size() > 3) {
+    if (components.size() < 1 || components.size() > 4) {
```

Comment:
`components.size() == 4` is only valid if that first element is `"default"`, right? What can happen if `components.size() == 4` but the first is not `"default"`?

Reply:
done